experiments:

  - name: tabletop_01
    goal: above_table_01
    pre_action: nothing
    post_action: nothing
    sound_bite: "how ya doin buddy?"

